### PR TITLE
Dev/mr relation

### PR DESCRIPTION
**Pull request by bakunyo**
MonthlyReport, MonthlyReportCommentの関連を追加

https://github.com/hr-dash/hr-dash/pull/66 を先に

### CODE REVIEW

File: spec/models/monthly_report_spec.rb [hunk 1]
```diff
@@ -19,58 +19,27 @@
...
-    let(:l_back) { 'mogu' }
-    let(:goals) { 'muga' }
-    let(:month) { 1 }
-    let(:year) { 2020 }
+    it { is_expected.to be_valid }
+    it { is_expected.to validate_presence_of(:user_id) }
+    it { is_expected.to validate_presence_of(:month) }
+    it { is_expected.to validate_inclusion_of(:month).in_range(1..12) }
+    it { is_expected.to validate_presence_of(:year) }
+    it { is_expected.to validate_numericality_of(:year).is_greater_than_or_equal_to(2000) }
```
Comment (freezer530000) [inline spec/models/monthly_report_spec.rb:28]: **[rubocop]**
- Line is too long. [91/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/models/monthly_report_spec.rb [hunk 2]
```diff
@@ -19,58 +19,27 @@
...
-      end
-      context 'invalid if month is out of 1 to 12 ' do
-        let(:month) { 0 }
-        it { expect(report).not_to be_valid }
-      end
-      context 'invalid if year is under 2000' do
-        let(:year) { 1999 }
-        it { expect(report).not_to be_valid }
+    describe 'Comments' do
+      let!(:comments) { create_list(:monthly_report_comment, 3, monthly_report: report) }
```
Comment (freezer530000) [inline spec/models/monthly_report_spec.rb]: **[rubocop]**
- Line is too long. [89/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/models/monthly_report_spec.rb [hunk 3]
```diff
@@ -19,58 +19,27 @@
...
+      it { expect(report.monthly_report_comments.size).to eq 3 }
+
+      context 'When report destroyed' do
+        before { report.destroy }
+        it { expect { comments.sample.reload }.to raise_error(ActiveRecord::RecordNotFound) }
```
Comment (freezer530000) [inline spec/models/monthly_report_spec.rb]: **[rubocop]**
- Line is too long. [93/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

**Comment by masahiro-yamazaki** [comment]
レビューします

**Comment by t-tonchim** [comment]
いいと思います！

**Comment by masataka-ohara** [comment]
見ます

**Comment by masataka-ohara** [comment]
確認しました

**Pull request merged**